cmd/to-crdgen/main.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019. PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	crdutils "github.com/ant31/crd-validation/pkg"
```

Comment:
Seems the code is not formatted by gofmt.

Reply:
I checked with gofmt and it seems already formatted.